VideoApp/VideoApp/TwilioVideo/LocalParticipant/LocalParticipantManager.swift

```diff
@@ -94,6 +94,11 @@ class LocalParticipantManager: NSObject, ObservableObject {
     func configure(identity: String) {
         self.identity = identity
     }
+    
+    func setHold(isOnHold: Bool) {
+        micTrack?.isEnabled = !isOnHold
+        cameraTrack?.isEnabled = !isOnHold
+    }
```

Comment:
I think this may need to be more complex to handle some edge cases but may create a separate ticket for that. For example what happens when a user changes mute status from inside the app when the video call is on hold. It could enable the track again if not careful.